webapp/app/js/compareperf.js

```diff
@@ -143,8 +144,17 @@ perf.controller('CompareResultsCtrl', [
       });
     }
 
-    var optionCollectionMap = {};
     $scope.dataLoading = true;
+    $scope.getDeltaClass = function(cr) { return cr.className; };
```

Comment:
I'm not sure about putting CSS logic inside the controller, but it doesn't look like angular provides better options (http://stackoverflow.com/questions/7792652/what-is-the-best-way-to-conditionally-apply-a-class). Could you at least consolidate these into one method like $scope.getCompareResultClasses() ? Just append the applicable classes into a list, then join them into a string and return it.